packages/scene-composer/src/utils/entityModelUtils/sceneUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { CreateEntityCommandInput } from '@aws-sdk/client-iottwinmaker';
+
+import { SCENE_ROOT_ENTITY_ID } from '../../common/entityModelConstants';
+import { getGlobalSettings } from '../../common/GlobalSettings';
+import { generateUUID } from '../mathUtils';
+
+export const createSceneEntityId = (sceneName: string) => {
+  return `SCENE_${sceneName}_${generateUUID()}`;
+};
+
+export const createSceneRootEntity = () => {
```

Comment:
Same?

Reply:
I'll add in my next PR if no other blocking comment